### DISCUSSION TITLE
verification email not arriving to my gmail account

### DISCUSSION THREAD

**Discussion by jots** [community/community#134891] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

trying to add an additional email to my account but the verification email is not arriving to my gmail account

**Comment by sabira-khan** [comment]
sometimes there's a typo in the email, maybe worth recheck? 
I'm pretty sure you've checked your junk / spam folders but hope you don't have any sort of filters turned on to filter out emails from x organization / outside x org etc.

**Comment by Njabulo240** [comment]
Make sure to check your current email inbox and don't forget to follow me.

**Comment by srijon57** [comment]
#Check Spam/Junk Folder: Sometimes, emails from new senders can end up in the spam or junk folder. Make sure to check there.

#Filters and Blocked Addresses: Ensure that you don't have any filters set up that might be redirecting these emails to another folder, or any addresses blocked that might be preventing the emails from arriving.

#Email Address: Double-check that you entered your email address correctly without any typos.

#Resend the Email: If possible, try to resend the verification email. There may have been a temporary issue when it was sent.

#Wait a Bit Longer: Occasionally, email delivery can be delayed due to server issues. Wait for some time and check again.

#Contact Support: If none of the above steps work, it might be helpful to contact the support team of the service from which you are trying to receive the verification email. They can check their server logs and provide further assistance.
